parser/proto.y

```diff
@@ -766,13 +765,11 @@ reservedNames : _RESERVED fieldNameStrings ';' {
 		$$ = ast.NewReservedIdentifiersNode($1.ToKeyword(), $2.idents, $2.commas, $3)
 	}
 
-reservedNamesWithEmptyDecls : _RESERVED fieldNameStrings semicolons {
-	  semi, extra := protolex.(*protoLex).requireSemicolon($3)
-		$$ = newNodeWithEmptyDecls(ast.NewReservedNamesNode($1.ToKeyword(), $2.names, $2.commas, semi), extra)
+reservedNamesWithEmptyDecls : _RESERVED fieldNameStrings ';' semicolons {
```

Comment:
Are the semicolons needed here to disambiguate? I would think they may only be needed for the reserved range production above.

Reply:
nice catch, removed!